package.json

```diff
@@ -31,7 +31,7 @@
     "lokijs": "1.5.6",
     "nativescript-urlhandler": "1.2.2",
     "p-queue": "3.0.0",
-    "pubnub": "4.21.6",
+    "pubnub": "4.21.7",
```

Comment:
Nope I installed the wrong version. I will need to change this.

Reply:
I have updated the `pubnub` dependency for the `kinvey-nativescript-sdk` to use the forked version since it is the only one that needs the forked version. All other packages can use the official released `pubnub` package.